streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -529,11 +531,13 @@ public void run() {
             cleanRun = true;
         } catch (final KafkaException e) {
             // just re-throw the exception as it should be logged already
+            // TODO: this is not true since task.update() call may throw ProcessorStateException
+            //       which is a KafkaException but it never get logged.
             throw e;
         } catch (final Exception e) {
             // we have caught all Kafka related exceptions, and other runtime exceptions
             // should be due to user application errors
-            log.error("{} Streams application error during processing: ", logPrefix, e);
+            log.error("{} Streams application error during processing", logPrefix, e);
```

Comment:
Note that log4j will add a space automatically between the message and the class path so it will be `suspending (org.apache[...])` even if we do not add a space ourselves.